arm/avhclient/aws_backend.py

```diff
@@ -820,21 +861,31 @@ def send_remote_command(self, command_list, working_dir, fail_if_unsuccess=True)
         This is a mandatory AVH backend method.
         """
         self._init()
-        logging.info(f"avh: command_list = {command_list}")
+        logging.debug("aws:command_list = %s", command_list)
         response = self.send_ssm_shell_command(
             command_list=command_list,
             working_dir=working_dir
         )
 
+        log_init = '='*80
+        logging.info(log_init) if enable_logging_info else logging.debug(log_init)
```

Comment:
This still causes a PyLint warning. Hence, I'd change this implementation to
```suggestion
        logging.log(logging.INFO if enable_logging_info else logging.DEBUG, '='*80)
```